department-of-veterans-affairs/caseflow
# TYLERB/SCT-Assign-Page: Added no limit client side queue tabs

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/JIRA-12345)

# Description
Added a method in queue_tab.rb named no_task_limit that allows a queue tab to set if it has a limit on tasks or not. This allows client side queue tabs to have more than 15 tasks if the method is overridden in the individual tabs.

## Acceptance Criteria

## Testing Plan

1. data setup
```ruby
FactoryBot.create_list(:specialty_case_team_assign_task, 30)
``` 
2. Login as an SCT user or SCT admin
3. Visit the sct bulk assign page either though the UI or the url /queue/SUPERUSER/assign?role=sct_coordinator
4. The queue should now be paginated and have more than 15 tasks
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/109369527/04266216-e821-4784-9d5f-e539c22d7b7b)
5. It should be sorted by docket number. This will only happen by default if the cached appeals job has been ran.


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
